feature-libs/user/account/styles/_verification-token-form.scss

```diff
@@ -0,0 +1,45 @@
+%cx-verification-token-form {
+  &.user-form {
+    .resend-link-text {
+      display: flex;
+      flex-direction: row;
+      width: 100%;
+      margin: auto;
+
+      .left-text {
+        padding: 0;
+        width: 50%;
+        text-align: var(--cx-text-align, left);
```

Comment:
Please remove all `--cx-text-align` variables. You can use standard text-align values. We are trying to clean up deprecated variables.

Reply:
fixed